_posts/2019-02-14-supervisor-strategies.md

```diff
@@ -0,0 +1,358 @@
+---
+author: Bobby Grayson
+author_link: https://github.com/notactuallypagemcconnell
```

Comment:
@tajchumber / @burden do we need this author link or is this now handled via the contributor's profile data?

Reply:
at the moment `author_link` is still required (see: https://github.com/elixirschool/elixirschool/blob/20bc771347cdfd6083d430ef6b88577df3b9586b/_layouts/post.html#L7)

but his can be easily fixed with by retrieving the data from `contributors.yml` instead of the post meta data 👍

Reply:
Thank you for the quick response @tajchumber! I think this would be a good improvement to make, keeping this information (the author bio, url, etc) in one place. What do you think?

Reply:
yeah it think it's best way to proceed.

There's one little problem though:
a lot of contributors don't have a `homepage`, what links should we display in case it's missing?
Should we fall back to `author_link` from the post metadata or link the contributor's github page?

Reply:
I think GitHub is a good fallback, that's what most author_links are 👍

Reply:
done!
https://github.com/elixirschool/elixirschool/pull/1720